### ISSUE TITLE
Code review 

### ISSUE THREAD

**Issue by ugur2889** [bounswe/bounswe2015group2#174]
Could you review  my code in here please ?   f2e12d765655a7c0842d7404ad89178309c572af

**Comment by CaferTayyar** [comment]
All right, I will review your code and give feedback.

**Comment by CaferTayyar** [comment]
-Code clarity: The code is understandable and there is no unclarity.

-Bugs:
Because there is a problem on back-end, recipes are not listed in dailyConsumption page. After solved the problem on back-end, the code will become better.

-Variable names: Variable names are understandable and clear for anybody who looks at the code.

-Indentation: indentation is suitable for conventional coding

-Comments: Comments in the code are very useful. What the code does can be understood easily when reading comments.

**Comment by TahaK** [comment]
* Code clarity: Asynchronous callback could be made as a seperate innerclass which could increase the clarity of the code.
* Bugs: I did not find any bugs
* Missing Features: Features have been implemented
* Variable names: Variable names are meaningful and adhere to the standards.
* Comments: The comments are predominantly inline comments it would be better if standard multi-line documentation is used especially in the beginning of methods.
* Code Analyser: Highlights the following : 
Spelling mistakes. Due to the use of Turkish words in the documentation
Declaration access. Variables could be declared as private

Overall, it is a successful implementation.
